src/ghci/mod.rs

```diff
@@ -707,6 +718,28 @@ impl Ghci {
         Ok(())
     }
 
+    /// `:unadd` a module or modules from the `ghci` session by path.
+    #[instrument(skip(self), level = "debug")]
+    async fn remove_modules(
+        &mut self,
+        paths: &[NormalPath],
+        log: &mut CompilationLog,
+    ) -> miette::Result<()> {
+        let modules = self.targets.format_modules(&self.search_paths, paths)?;
```

Comment:
Each `:unadd` implicitly reloads as well, so we have to `:unadd` all the modules in a single command so that GHCi doesn't try to load a bunch of removed modules after each one.

Reply:
What you said should probably be a comment in the code